tap_s3_csv/conversion.py

```diff
@@ -32,7 +32,15 @@ def infer_column(column, dateFormatMap, lengths):
 
 def infer_number(column):
     tmpCol = column.copy()
-    tmpResult = pd.to_numeric(tmpCol, errors='ignore')
+    tmpCol = tmpCol.apply(lambda x: x.replace(',', ''))
```

Comment:
Not sure if we want to fix this now. There's another ticket where `,` is used as decimal separator in some locale settings and we would require some design change decision. I guess it would be best to check with Ganesh first?
https://varicent.atlassian.net/browse/WP-21230

@ChrisLing1

Reply:
If we want to support different locale, we can make that change later. IMO we should just stick to what we've always expected in terms of numbers.